js/pages/estimation/components/cca-specification-view-edit.html

```diff
@@ -27,15 +27,15 @@
                         <div class="panel panel-default">
                             <div class="panel-heading">
                             <i class="fa fa-users"></i> Comparisons
-                            <button type="button" class="btn btn-primary btn-sm pull-right" data-bind="click: addComparison"><i class="fa fa-plus"></i> Add Comparison</button>
+                            <button type="button" class="btn btn-primary btn-sm pull-right" data-bind="click: addComparison, disable: !$component.isEditPermitted()"><i class="fa fa-plus"></i> Add Comparison</button>
                             </div>
                             <div class="panel-body">
                                 <faceted-datatable params="{ 
-                                columns: options.cca.comparisonTableColumns, 
-                                options: options.cca.comparisonTableOptions, 
-                                pageLength: options.cca.comparisonTableOptions.pageLength,
-                                lengthMenu: options.cca.comparisonTableOptions.lengthMenu,
-                                dom: options.cca.comparisonTableOptions.dom,
+                                columns: $component.cca.comparisonTableColumns, 
```

Comment:
pls, omit the $component prefix when one's not required

Reply:
fixed